extensions/roc-package-web-app-react/views/roc-package-web-app-react/main.njk

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html {{ head.htmlAttributes.toString() | safe }}>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="x-ua-compatible" content="ie=edge">
+	{{ head.title.toString() | safe }}
+	{{ head.base.toString() | safe }}
+	{{ head.meta.toString() | safe }}
+
+	{% block head %}{% endblock %}
+
+	{{ head.script.toString() | safe }}
+	{{ head.link.toString() | safe }}
+	{{ head.style.toString() | safe }}
+
+	{% if dist %}
+		<link rel="stylesheet" href="{{ styleName }}">
+	{% endif %}
+</head>
+{% if custom.bodyClass %}
+<body class="{{ custom.bodyClass | safe }}">
+{% else %}
+<body {{ head.bodyAttributes.toString() | safe }}>
+{% endif %}
+
+{% block bodyPreApplication %}{% endblock %}
```

Comment:
Sure. I'm also thinking that just `preApplication` and `postApplication` is enough?